mailet/base/src/test/java/org/apache/mailet/base/test/MimeMessageBuilder.java

```diff
@@ -305,22 +325,21 @@ public MimeMessage build() throws MessagingException {
         Preconditions.checkState(!(text.isPresent() && content.isPresent()), "Can not get at the same time a text and a content");
         MimeMessage mimeMessage = new MimeMessage(Session.getInstance(new Properties()));
         if (text.isPresent()) {
-            BodyPart bodyPart = new MimeBodyPart();
-            bodyPart.setText(text.get());
-            mimeMessage.setContent(bodyPart, "text/plain");
+            mimeMessage.setContent(text.get(), "text/plain");
```

Comment:
I don't understand your comment. This code perfectly instanciate a MimeMessage with a simple text body.

Here I define I want a text/plain body. When needed we will handle logic for binary bodies. Anyway, I believe it is out of the scope of what I am doing.